services/motion/builtin/builtin_test.go

```diff
@@ -386,8 +386,6 @@ func TestInjectedMoveOnGlobe(t *testing.T) {
 	destGP := geo.NewPoint(gp.Lat(), gp.Lng()+0.0000009)
 
 	t.Run("ensure success to a nearby geo point", func(t *testing.T) {
-		t.Parallel()
```

Comment:
Rather than removing these, can we figure out which piece has a race and put the instantiation of that into the `t.Run()`?

Reply:
solution was to add mutex lock to injected movement sensor position function only.